api/main_endpoints/routes/User.js

```diff
@@ -250,6 +251,39 @@ router.get('/callback', async function(req, res) {
     });
 });
 
+router.post('/getUserFromDiscordId', (req, res) => {
```

Comment:
since we capitalize ID in discordID, we could do the same capitalization here. up to you